Mage.Sets/src/mage/cards/f/FeralEncounter.java

```diff
@@ -0,0 +1,88 @@
+package mage.cards.f;
+
+import mage.abilities.Ability;
+import mage.abilities.DelayedTriggeredAbility;
+import mage.abilities.common.delayed.AtTheBeginOfCombatDelayedTriggeredAbility;
+import mage.abilities.effects.common.CreateDelayedTriggeredAbilityEffect;
+import mage.abilities.effects.common.DamageWithPowerFromOneToAnotherTargetEffect;
+import mage.abilities.effects.common.LookLibraryAndPickControllerEffect;
+import mage.cards.Card;
+import mage.cards.CardImpl;
+import mage.cards.CardSetInfo;
+import mage.cards.Cards;
+import mage.constants.*;
+import mage.filter.StaticFilters;
+import mage.filter.common.FilterCreaturePermanent;
+import mage.game.Game;
+import mage.players.Player;
+import mage.target.common.TargetControlledCreaturePermanent;
+import mage.target.common.TargetCreaturePermanent;
+import mage.util.CardUtil;
+
+import java.util.UUID;
+
+/**
+ * @author Susucr
+ */
+public final class FeralEncounter extends CardImpl {
+
+    private static final FilterCreaturePermanent filter = new FilterCreaturePermanent("creature you don't control");
+
+    static {
+        filter.add(TargetController.NOT_YOU.getControllerPredicate());
+    }
+
+    public FeralEncounter(UUID ownerId, CardSetInfo setInfo) {
+        super(ownerId, setInfo, new CardType[]{CardType.SORCERY}, "{G}{G}");
+
+        // Look at the top five cards of your library. You may exile a creature card from among them. Put the rest on the bottom of your library in a random order. You may cast the exiled card this turn. At the beginning of the next combat phase this turn, target creature you control deals damage equal to its power to up to one target creature you don't control.
+        this.getSpellAbility().addEffect(new FeralEncounterEffect());
+
+        DelayedTriggeredAbility delayed = new AtTheBeginOfCombatDelayedTriggeredAbility(
+                new DamageWithPowerFromOneToAnotherTargetEffect(),
+                AtTheBeginOfCombatDelayedTriggeredAbility.PhaseSelection.NEXT_COMBAT_PHASE_THIS_TURN
+        );
+        delayed.addTarget(new TargetControlledCreaturePermanent());
+        delayed.addTarget(new TargetCreaturePermanent(0, 1, filter, false));
+        this.getSpellAbility().addEffect(new CreateDelayedTriggeredAbilityEffect(delayed)
+                .setText("At the beginning of the next combat phase this turn, target creature you "
```

Comment:
curious why this set text is necessary?

Reply:
Because the delayed trigger was missing a capitalized A to At, but changed that in its constructor.